src/pages/workspace/categories/CategorySettingsPage.tsx

```diff
@@ -55,8 +56,10 @@ function CategorySettingsPage({
     const policyCurrency = policy?.outputCurrency ?? CONST.CURRENCY.USD;
     const policyCategoryExpenseLimitType = policyCategory?.expenseLimitType ?? CONST.POLICY.EXPENSE_LIMIT_TYPES.EXPENSE;
 
+    const [isCannotDisableLastCategoryModalVisible, setIsCannotDisableLastCategoryModalVisible] = useState(false);
     const areCommentsRequired = policyCategory?.areCommentsRequired ?? false;
     const isQuickSettingsFlow = !!backTo;
+    const shouldPreventDisable = policyCategory?.enabled && policy?.requiresCategory && getEnabledCategoriesCount(policyCategories) === 1;
```

Comment:
```suggestion
    const shouldPreventDisable = policy?.requiresCategory && getEnabledCategoriesCount(policyCategories) === 1;
```
NAB but I think this is more logical and the "enabled" condition is already part of `updateWorkspaceCategoryEnabled`. We just need to add it to the `showLockIcon` as well